horizon/config.py

```diff
@@ -103,6 +104,24 @@ def __new__(cls, prefix=None, is_model=True):
     # enable datadog APM tracing
     ENABLE_MONITORING = confi.bool("ENABLE_MONITORING", False)
 
+    ENABLE_OFFLINE_MODE = confi.bool(
+        "ENABLE_OFFLINE_MODE",
+        False,
+        description="if true, sidecar will use a file backup to restore configuration and policy data when cloud services are unavailable",
+    )
+
+    OFFLINE_MODE_BACKUP_PATH = confi.str(
+        "OFFLINE_MODE_BACKUP_PATH",
+        "/app/backup/sidecar_config.json",
```

Comment:
I think it is better to make this configuration be the directory instead the file itself and make "sidecar_config.json" hard-coded same way "policy_store_data.json"